Src/GoogleApis.Auth.Tests/OAuth2/Flows/AuthorizationCodeFlowTests.cs

```diff
@@ -76,10 +76,6 @@ public void TestConstructor_DefaultValues()
             Assert.NotNull(flow.HttpClient);
             Assert.NotNull(flow.Scopes);
             Assert.That(flow.TokenServerUrl, Is.EqualTo("https://token.com"));
-
-            Assert.That(flow.HttpClient.MessageHandler.UnsuccessfulResponseHandlers.Count(), Is.EqualTo(1));

```

Comment:
Since we annotated the UnsuccessfulResponseHandlers property as obsolete, there's not actually a supported way to query the list. I think that's good -- I can't think of great reasons for people to do it -- but it also means there's no supported way to perform these tests.

I thought the best solution was to remove them, since they test implementation rather than behavior. Admittedly, I haven't verified that the *behavior* is tested anywhere.